wcfsetup/install/files/lib/system/WCF.class.php

```diff
@@ -813,6 +814,10 @@ protected function assignDefaultTemplateVariables()
             }
         }
 
+        self::getTPL()->assign([
+            '__phrasePreloadCacheUrl' => PhrasePreloadCache::getUrl(self::getLanguage()),
+        ]);
```

Comment:
Please do not add new logic to `wcf\system\WCF`. WCF should perform bootstrapping only. I've previously moved the registration of the cronjob template variable to the bootstrap file to keep WCF clean.

In this case the registration in WCF is especially bad, because it relies on the language and thus a request-specific value. In the long run this might even need to be moved into a middleware.